kleros/website
# feat(header): new products dropdown

# Description

This PR implements an improved list of product badges for the header. The new list allows you to hover over badges to see a description of the product.

<img width="1782" alt="Screen Shot 2020-10-22 at 9 38 13 AM" src="https://user-images.githubusercontent.com/19157096/96879975-a72fdf80-144a-11eb-8ee7-6d2f1efe6b89.png">

We may also want to use this for the products section under the fold on the main page and somewhere on the integrations page.

Web3's `scrypt` was causing problems, so I took the opportunity to update Web3 to a newer version and fixed all the linting errors while I was at it.

# Rationale

The code lives in a `ProductBadges` component to get rid of the current duplication of the badge code and leverage this new functionality wherever we want to.

Framer Motion powers the animations. I prototyped this in Framer X, and the library is the best right now for things like this.

I followed all project's patterns, including internationalization, except for the CSS.

The CSS changes in this PR provide a much better example of how to leverage CSS modules correctly. The current code relies almost entirely on DOM node ordering because it heavily overuses nested selectors and similar tools. That makes it hard to make markup or style changes without breaking lots of unrelated code.

# How To Test This?

Make sure nothing regressed due to the Web3 update and try out the new products dropdown in the navigation bar.
